submissions/dafen173/pop-up2/index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="uk">
+	<head>
+		<title>PopUp menu</title>
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="style.css" />
```

Comment:
The single tags should be without slashes, according to HTML5 specification. Could you please remove all slashes where they were added?